src/Display/Tooltip/Tooltip.tsx

```diff
@@ -15,30 +15,38 @@ const Tooltip: React.FunctionComponent<Props> = ({
   position,
   ...defaultProps
 }) => {
-  const ref = React.useRef(null);
-  const [isHover, setIsHover] = React.useState(false);
-  const showTooltip = () => setIsHover(true);
-  const hideTooltip = () => setIsHover(false);
+  const contentRef = React.useRef(null);
+  const [isShow, setIsShow] = React.useState(false);
+  const showTooltip = () => setIsShow(true);
+  const hideTooltip = () => setIsShow(false);
 
-  const touchOutside = (event: TouchEvent) => {
-    const element = event.target as HTMLElement;
-    const hasTouchedOutside = !ref.current.contains(element);
+  const hideTooltipIfTouchOutside = React.useCallback(
+    (event: TouchEvent | React.TouchEvent) => {
+      const hasTouchedOutsideOfTooltipContent = !contentRef.current.contains(
```

Comment:
in order to prevent null pointer, suggest to add check for contentRef.current

Reply:
Since `hideTooltipIfTouchOutside` muse be triggered after `showTooltip`, don't think that check is necessary.